docs/resources/worker_script.md

```diff
@@ -39,6 +39,12 @@ resource "cloudflare_worker_script" "my_script" {
     name = "MY_EXAMPLE_WASM"
     module = filebase64("example.wasm")
   }
+
```

Comment:
this needs to go to the `templates` directory and then re-run `make docs` to have it populate this file.